packages/app/src/app/pages/Dashboard/Components/Branch/BranchCard.tsx

```diff
@@ -53,7 +51,7 @@ export const BranchCard: React.FC<BranchDetails> = ({
         align="center"
         justify="center"
       >
-        <Icon aria-hidden color="#808080" name="fork" size={40} />
+        <Icon color="#808080" name="branch" size={40} />
```

Comment:
@alexnm why are we removing the `aria-hidden` from the Icon?

Reply:
there's a `aria-hidden` on the `Icon` component added yesterday or two days ago